### DISCUSSION TITLE
Using Prism + WPF with .Net8

### DISCUSSION THREAD

**Discussion by YkTru** [PrismLibrary/Prism#2924] [WPF] [answered]
Since it is no longer part of the .Net foundation, is it risky to use the most recent package for Prism with .Net8? 
And will Prism "die" as Caliburn Micro did eventually? Thank you

**Comment by brianlagunas** [comment, marked as answer]
Just so you know, the .NET Foundation has nothing to do with whether a project is compatible with the latest and greatest versions of .NET or if it will be maintained forever.

Yes, you can use Prism with .NET 8. No, Prism will not "die", the activity on the recent PRs should be proof of that. We are actually preparing for a massive Prism 9 release.

**Reply by YkTru** [reply]
Great, thank you very much!
